tests/Presentation/RangeFacetHtmlBuilderTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseFacetedSearch\Tests\Presentation;
+
+use PHPUnit\Framework\TestCase;
+use ProfessionalWiki\WikibaseFacetedSearch\Application\FacetConfig;
+use ProfessionalWiki\WikibaseFacetedSearch\Application\FacetType;
+use ProfessionalWiki\WikibaseFacetedSearch\Application\PropertyConstraints;
+use ProfessionalWiki\WikibaseFacetedSearch\Presentation\RangeFacetHtmlBuilder;
+use ProfessionalWiki\WikibaseFacetedSearch\WikibaseFacetedSearchExtension;
+use Wikibase\DataModel\Entity\ItemId;
+use Wikibase\DataModel\Entity\NumericPropertyId;
+
+/**
+ * @covers \ProfessionalWiki\WikibaseFacetedSearch\Presentation\RangeFacetHtmlBuilder
+ */
+class RangeFacetHtmlBuilderTest extends TestCase {
+
+	private const FACET_PROPERTY_ID = 'P42';
+
+	public function testRendersTemplateWithMinAndMax(): void {
+		$html = $this->newRangeFacetHtmlBuilder()->buildHtml(
+			config: $this->newConfig(),
+			state: $this->newPropertyConstraints()->withInclusiveMinimum( 1337 )->withInclusiveMaximum( 9001 )
+		);
+
+		//$this->assertStringContainsString( self::FACET_PROPERTY_ID, $html );
```

Comment:
Leftover code? Or is something not working here?

Reply:
The ID is not present yet in the HTML. I left this as a comment since I assume it will be added at some point